tracecat/config.py

```diff
@@ -106,3 +106,13 @@
 RETRY_MIN_WAIT_TIME = 1
 RETRY_MAX_WAIT_TIME = 60
 RETRY_STOP_AFTER_ATTEMPT = 5
+
+# SMTP Settings
+SMTP_HOST = os.environ.get("SMTP_HOST", "")
+SMTP_PORT = os.environ.get("SMTP_PORT", 25)
+SMTP_STARTTLS = os.environ.get("SMTP_STARTTLS", False)
+SMTP_SSL = os.environ.get("SMTP_SSL", False)
+SMTP_IGNORE_CERT_ERRORS = os.environ.get("SMTP_IGNORE_CERT_ERRORS", False)
+SMTP_AUTH = os.environ.get("SMTP_AUTH", False)
```

Comment:
Ah I might have not been too clear about this -- for these boolean env vars you're right they're typically stored as either "1" or "0" (some string value), which is why we'd do something like this to transform it into a boolean:
```python
SMTP_SSL = os.getenv("SMTP_SSL", "0").lower() in ("1", "true") # Defaults to `false`
# Same for SMTP_STARTTLS, SMTP_IGNORE_CERT_ERRORS, SMTP_AUTH
```
so that you can assign any of "true" (case insensitive) or "1" to the undelying env var to make it `true` and anything else will make it `false`. The way you've got it right now, checking the truthiness of the config var all strings will resolve to `true`.

Also, can we also make the names more descriptive so its more obvious that it's a boolean value and not some other kind of config - i.e.
`SMTP_AUTH` -> `SMTP_AUTH_ENABLED`
`SMTP_STARTTLS` -> `SMTP_STARTTLS_ENABLED`
`SMTP_SSL` -> `SMTP_SSL_ENABLED`

Reply:
@daryllimyt Do you want the names updated in the .env file as well?

Reply:
yes please, thanks!

Reply:
> yes please, thanks!

done